LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2015–2022, Miriam Suzanne.
+Copyright (c) 2015–2026, OddBird LLC.
```

Comment:
@mirisuzanne I don't feel strongly about this change -- was updating the date and noticed most of our projects are licensed to OddBird 🤷